README.md

```diff
@@ -5,3 +5,9 @@
 - This one is correct
 - This needs a fix
 - This one is correct too
+
+## Another List of stuff
+
+- This one is correct
+* This needs a fix
```

Comment:
Codacy has a fix for the issue: [Expected: dash; Actual: asterisk](https://app.codacy.com/gh/troubleshoot-codacy/markdown-fixes/pullRequest?prid=7586305)
```suggestion
- This needs a fix
```